src/Operators/divergence_operators.jl

```diff
@@ -36,39 +22,13 @@ end
 
 Calculates the 2D divergence ∂x u + ∂y v via
 
-    1/V * [δxᶜᵃᵃ(Ax * u) + δyᵃᶜᵃ(Ay * v)]
+    1/Azᶜᶜᵃ * [δxᶜᵃᵃ(Δy * u) + δyᵃᶜᵃ(Δx * v)]
 
-which will end up at the location `cca`.
+where `Azᶜᶜᵃ` is the area of the cell centered on (Center, Center, Any) --- a tracer cell,
+`Δy` is the length of the cell centered on (Face, Center, Any) in `y` (a `u` cell),
+and `Δx` is the length of the cell centered on (Center, Face, Any) in `x` (a `v` cell).
+`div_xyᶜᶜᵃ` ends up at the location `cca`.
 """
-@inline function div_xyᶜᶜᵃ(i, j, k, grid, u, v)
-    return 1/Vᵃᵃᶜ(i, j, k, grid) * (δxᶜᵃᵃ(i, j, k, grid, Ax_ψᵃᵃᶠ, u) +
-                                    δyᵃᶜᵃ(i, j, k, grid, Ay_ψᵃᵃᶠ, v))
-end
-
-"""
-    div_xzᶜᶜᵃ(i, j, k, grid, u, w)
-
-Calculates the 2D divergence ∂x u + ∂z w via
-
-    1/V * [δxᶜᵃᵃ(Ax * u) + δzᵃᵃᶜ(Az * w)]
-
-which will end up at the location `cac`.
-"""
-@inline function div_xzᶜᵃᶜ(i, j, k, grid, u, w)
-    return 1/Vᵃᵃᶜ(i, j, k, grid) * (δxᶜᵃᵃ(i, j, k, grid, Ax_ψᵃᵃᶠ, u) +
-                                    δzᵃᵃᶜ(i, j, k, grid, Az_ψᵃᵃᵃ, w))
-end
-
-"""
-    div_yzᶜᶜᵃ(i, j, k, grid, v, w)
-
-Calculates the 2D divergence ∂y v + ∂z w via
-
-    1/V * [δyᶜᵃᵃ(Ay * v) + δzᵃᵃᶜ(Az * w)]
-
-which will end up at the location `acc`.
-"""
-@inline function div_yzᵃᶜᶜ(i, j, k, grid, v, w)
-    return 1/Vᵃᵃᶜ(i, j, k, grid) * (δyᵃᶜᵃ(i, j, k, grid, Ay_ψᵃᵃᶠ, v) +
-                                    δzᵃᵃᶜ(i, j, k, grid, Az_ψᵃᵃᵃ, w))
-end
+@inline div_xyᶜᶜᵃ(i, j, k, grid, u, v) =
```

Comment:
Very nice that it's more compact and I suppose an answer to my previous question will help me to  understand why this works so nicely